libkbfs/delayed_cancellation_test.go

```diff
@@ -102,6 +102,25 @@ func TestDelayedCancellationCleanupWhileNotEnabled(t *testing.T) {
 	}
 }
 
+func TestDelayedCancellationSecondEnable(t *testing.T) {
+	t.Parallel()
+
+	ctx, cancel := makeContextWithDelayedCancellation(t)
+	defer cancel()
+
+	err := EnableDelayedCancellationWithGracePeriod(ctx, 0)
+	if err != nil {
+		t.Fatalf("1st EnableDelayedCancellationWithGracePeriod failed: %v", err)
+	}
+	time.Sleep(5 * time.Millisecond) // make sure async ops sorts out
```

Comment:
I'm ok with removing this test, because it don't test anything.  The sleep is meaningless because there is nothing asynchronous that matters for the test.

But I think if you changed the test to one that canceled the context, and then made sure the second call got a proper error, that would be a worthwhile test.